src/views/Login.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="min-h-screen bg-gray-50 flex flex-col justify-center py-12 sm:px-6 lg:px-8">
+    <div class="sm:mx-auto sm:w-full sm:max-w-md">
+      <img class="mx-auto h-12 w-auto" src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg" alt="Workflow" />
+      <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-800">
+         {{ $t('register.button.label.signIn') }}
+      </h2>
+    </div>
+    <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
+      <div class="bg-white py-8 px-4 shadow sm:rounded-lg sm:px-10">
+           <ValidationObserver v-slot="{ valid }">
+        <form class="space-y-6" action="#" method="POST" @submit.prevent="handleSubmit">
+          <div>
+            <div class="mt-1">
```

Comment:
Why do you have the nested divs here, they can be removed, and style the input or add a wrapper div on every input if you want to set the container width